### ISSUE TITLE
Vimeo's New Privacy Hash Just Broke Plyr (Today)

### ISSUE THREAD

**Issue by jhakooz** [sampotts/plyr#2672]
### Expected behaviour
Plyr can be used to play Vimeo videos automatically by providing **data-plyr-provider** and **data-plyr-embed-id** attributes to a div. 

### Actual behaviour
New videos uploaded to Vimeo now must include a privacy hash which is appended to the video id. For example:
Old Vimeo Id: 823098123
New Vimeo Id w/ Privacy Hash: 823098123?h=96ffdf03e6

The addition of a question mark in the new Vimeo Id w/ privacy hash breaks Plyr since Plyr automatically adds a question mark before appending parameters to the URL. 

This is pretty easy to fix by manually changing the Plyr source code to use an ampersand instead of a question mark. However, things get messy when you are trying to stream old videos and new videos with the privacy hash. 

### Steps to reproduce
Try adding a newly uploaded video id with privacy hash to the **data-plyr-embed-id** attribute. You will get a Vimeo error. 

### Environment
All 

### Bottom Line
I realize that Plyr is merging with Vidstack and development may have stopped for Plyr, but this is about to break a lot of sites that use Plyr and Vimeo. And since the fix may be rather simple, I thought I would let you know.

Thanks for everything you've done with this project!

**Comment by domas838** [comment]
Workaround

https://github.com/sampotts/plyr/issues/2321#issuecomment-921088437

**Comment by ebrahimMaher** [comment]
same issue...

**Comment by jhakooz** [comment]
This just in... after 3 days Vimeo announced the latest privacy update had "unforeseen issues" and they rolled back the update. 
Wonder if they'll pay for my heart meds after all that?
